app/models/mixins/miq_web_server_worker_mixin.rb

```diff
@@ -137,6 +137,7 @@ def start
   end
 
   def terminate
+    require 'miq-process'
```

Comment:
Why add this here? Was it causing a problem?

Reply:
Yes, I needed `terminate` to run and delete the row and `MiqProcess.alive?` is called on line 149.

Reply:
Note, this was a test only change that I also put into the test but it seemed strange to reference `miq-process` since it wouldn't autoload from the gem.  I believe this code was written when `MiqProcess` was in core and would autoload out of the `lib` directory.